### PR TITLE
[new release] thread-local-storage (0.1)

### PR DESCRIPTION
**Pull request by c-cube**
Efficient thread local storage for OCaml

- Project page: <a href="https://github.com/c-cube/thread-local-storage">https://github.com/c-cube/thread-local-storage</a>
- Documentation: <a href="https://c-cube.github.io/thread-local-storage">https://c-cube.github.io/thread-local-storage</a>

##### CHANGES:

initial release

### CODE REVIEW

**Comment by c-cube** [comment]
(I don't want to require 2.7 if I can avoid it. Can we just forget about the `dune subst`? It's literally not used at all by the package)

File: packages/thread-local-storage/thread-local-storage.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+tags: ["topics" "threads" "tls"]
+homepage: "https://github.com/c-cube/thread-local-storage"
+doc: "https://c-cube.github.io/thread-local-storage"
+bug-reports: "https://github.com/c-cube/thread-local-storage/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.0"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/thread-local-storage/thread-local-storage.0.1/opam:15]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by mseri** [comment]
Yes don’t worry. That is easy to fix manually before merging

**Comment by c-cube** [comment]
Thank you :)

**Pull request merged**